api/v1beta1/nodemodulesconfig_types.go

```diff
@@ -47,9 +54,14 @@ type NodeModulesConfigSpec struct {
 }
 
 type NodeModuleStatus struct {
-	Name       string `json:"name"`
-	Namespace  string `json:"namespace"`
-	KmodLoaded bool   `json:"kmodLoaded"`
+	//+optional
+	Config *ModuleConfig `json:"config,omitempty"`
+	//+optional
+	LastTransitionTime *metav1.Time     `json:"lastTransitionTime,omitempty"`
+	Name               string           `json:"name"`
+	Namespace          string           `json:"namespace"`
+	ServiceAccountName string           `json:"serviceAccountName"`
```

Comment:
why the ServiceAccount is not part of the Config, if InsecurePull is?

Reply:
InsecurePull is part of the worker app config because it describes how to pull the image. ServiceAccountName is not part of the config because the worker app does not need it.